snapcraft/plugins/kernel.py

```diff
@@ -387,6 +387,25 @@ def _parse_kernel_release(self):
                 )
             )
 
+    def _get_kernel_version(self):
+        kernelversion = (
+            subprocess.check_output("make kernelversion", shell=True, cwd=self.builddir)
+            .decode("utf-8")
+            .strip()
+        )
+        if not kernelversion:
+            raise ValueError("make kernelversion didn't produce any version")
```

Comment:
you need to raise an exception Inheriting from `SnapcraftError`. Look at the `dump` plugin for an example. Missing a new sentence on how to fix the issue.